cmd/sfncli/sfncli.go

```diff
@@ -126,9 +126,12 @@ func main() {
 			log.Info("getactivitytask-stop")
 		default:
 			cw.SetActiveState(false)
+			cw.SetPaused(true)
```

Comment:
can we add a comment explaining the general context of not counting paused time towards inactive time?